commands/conduit_cli.py

```diff
@@ -0,0 +1,84 @@
+import os
+import subprocess
+import time
+
+import boto3
+import click
+
+from commands.utils import check_aws_conn
+
+
+def get_cluster_arn(app, env):
+    client = boto3.client("resourcegroupstaggingapi")
+
+    response = client.get_resources(
+        TagFilters=[
+            {"Key": "copilot-application", "Values": [app]},
+            {"Key": "copilot-environment", "Values": [env]},
+            {"Key": "aws:cloudformation:logical-id", "Values": ["Cluster"]},
+        ]
+    )
+
+    return response["ResourceTagMappingList"][0]["ResourceARN"]
+
+
+def create_task(app, env, secret_arn):
+    command = f"copilot task run -n dbtunnel --dockerfile Dockerfile --secrets DB_SECRET={secret_arn} --app {app} --env {env}"
+    subprocess.call(command, shell=True)
+
+
+def get_postgres_secret_arn(app, env):
+    secret_name = f"/copilot/{app}/{env}/secrets/POSTGRES"
+
+    return boto3.client("secretsmanager").get_secret_value(SecretId=secret_name)["ARN"]
+
+
+def is_task_running(cluster_arn):
+    tasks = boto3.client("ecs").list_tasks(cluster=cluster_arn, desiredStatus="RUNNING", family="copilot-dbtunnel")
+
+    try:
+        return tasks["taskArns"]
+    except ValueError:
+        return False
+
+
+def exec_into_task(app, env, cluster_arn):
+    # There is a delay between a task's being created and its health status changing from PROVISIONING to RUNNING,
+    # so we need to wait before running the exec command or timeout if taking too long.
+    timeout = time.time() + 60
+    connected = False
+    while time.time() < timeout:
+        if is_task_running(cluster_arn):
+            os.system(f"copilot task exec --app {app} --env {env}")
+            connected = True
+            break
+
+    if connected == False:
```

Comment:
```suggestion
    if not connected:
```

Reply:
I think what I have currently is more explicit. I don't want to return an empty string or list and, if I did by accident, I'd want the test to complain

Reply:
That's fair.

I only just found out because of this that `==` seems to be an exact comparison in Python, equivalent to `===` in most other languages I have used.

I wonder if it's worth type hinting this code?

Reply:
Ah yep, I forgot to add type hinting. Will do that now, although I've just realised it won't help in this particular instance because the function doesn't return anything

Reply:
In fact, as my brain is starting to spin up better, I've realised that `connected` can only ever be  `True` or `False` in this context, so maybe my attempt to suggest something more "pythonic" is still valid?